spec/models/abilities/administrator_spec.rb

```diff
@@ -20,6 +20,10 @@
   let(:future_poll) { create(:poll, :future) }
   let(:current_poll_question) { create(:poll_question) }
   let(:future_poll_question) { create(:poll_question, poll: future_poll) }
+  let(:future_poll_question_open) { create(:poll_question_open, poll: future_poll) }
+  let(:future_poll_question_option_open) do
+    create(:poll_question_option, question: future_poll_question_open)
+  end
```

Comment:
Since we don't allow creating options for open-ended questions, what do you think about building a new record without saving it in the database?

```suggestion
  let(:future_poll_question_option_open) { future_poll_question_open.question_options.new }
```